src/ServiceControl.Monitoring/Http/Diagrams/TimingsAggregator.cs

```diff
@@ -111,6 +64,7 @@ public class MonitoredEndpoint
         public string[] EndpointInstanceIds { get; set; }
         public MonitoredEndpointValues ProcessingTime { get; set; }
         public MonitoredEndpointValues CriticalTime { get; set; }
+        public MonitoredEndpointValues Retries { get; set; }
```

Comment:
Yeah. This is certainly the intent. That change needs to be done in conjunction with the UI though as it will affect the UI.